pkg/tfgen/generate_schema.go

```diff
@@ -553,6 +554,18 @@ func (g *schemaGenerator) genProperty(prop *variable) pschema.PropertySpec {
 			if i, ok := defaultValue.(int); ok {
 				defaultValue = float64(i)
 			}
+			rt := reflect.TypeOf(defaultValue)
+			if rt != nil {
+				switch rt.Kind() {
+				case reflect.Array, reflect.Slice, reflect.Map, reflect.Struct:
```

Comment:
Nit: once we are in here we can complete the case for all possibilities from the docs:

const (
 Invalid Kind = iota
 Bool
 Int
 Int8
 Int16
 Int32
 Int64
 Uint
 Uint8
 Uint16
 Uint32
 Uint64
 Uintptr
 Float32
 Float64
 Complex64
 Complex128
 Array
 Chan
 Func
 Interface
 Map
 Pointer
 Slice
 String
 Struct
 UnsafePointer
)

Unlikely to be seeing UnsafePointer in practice but it is technically not supported :)